cl/api/urls.py

```diff
@@ -179,30 +179,110 @@
         views.citation_lookup_api,
         name="citation_lookup_api",
     ),
+    path(
+        "help/api/rest/v3/case-law/",
+        TemplateView.as_view(
+            template_name="case-law-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="case_law_api_help",
+    ),
+    path(
+        "help/api/rest/v3/citations/",
+        TemplateView.as_view(
+            template_name="citation-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="citation_api_help",
+    ),
+    path(
+        "help/api/rest/v3/pacer/",
+        TemplateView.as_view(
+            template_name="pacer-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="pacer_api_help",
+    ),
+    path(
+        "help/api/rest/v3/recap/",
+        TemplateView.as_view(
+            template_name="recap-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="recap_api_help",
+    ),
+    path(
+        "help/api/rest/v3/judges/",
+        TemplateView.as_view(
+            template_name="judge-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="judge_api_help",
+    ),
+    path(
+        "help/api/rest/v3/oral-arguments/",
+        TemplateView.as_view(
+            template_name="oral-argument-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="oral_argument_api_help",
+    ),
+    path(
+        "help/api/rest/v3/visualizations/",
+        TemplateView.as_view(
+            template_name="visualizations-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
+        name="visualization_api_help",
+    ),
     path(
         "help/api/rest/v3/financial-disclosures/",
-        views.financial_disclosures_api_help_help,
+        TemplateView.as_view(
+            # "financial-disclosure-docs-vlatest.html"
+            template_name="financial-disclosure-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
         name="financial_disclosures_api_help",
     ),
     path(
         "help/api/rest/v3/search/",
-        views.search_api_help,
+        TemplateView.as_view(
+            template_name="search-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
         name="search_api_help",
     ),
     path(
         "help/api/rest/v3/alerts/",
-        views.alert_api_help,
+        TemplateView.as_view(
+            template_name="alert-api-docs-vlatest.html",
+            extra_context={"private": False},
+        ),
         name="alert_api_help",
     ),
+    path(
+        "help/api/rest/v3/fields/",
+        TemplateView.as_view(
+            template_name="field-help.html",
+            extra_context={"private": True},
```

Comment:
im not sure why this is set private=True

Reply:
I don't think anybody would really want to google this.